docs/source/guides/statistical_insights.ipynb

```diff
@@ -152,6 +152,33 @@
     "mi = df.ww.mutual_information(include_index=True)\n",
     "mi[mi['column_1'].isin(['order_product_id']) | mi['column_2'].isin(['order_product_id'])]"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Outlier Detection\n",
+    "- `Seriew.ww.outlier_dict`\n",
+    "- `Series.ww.box_plot_dict`\n",
+    "\n",
+    "Woodwork allows for univariate outlier detection using the IQR, or interquartile range, method. This can be done on a by-column basis using `series.ww.outliers_dict` method, which will just calculate the outliers, or the `series.ww.box_plot_dict` method that also includes the statistical data necessary for building a box and whisker plot."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "total = df.ww['total']\n",
+    "box_plot_dict = total.ww.box_plot_dict()\n",
+    "\n",
+    "print('high bound: ', box_plot_dict['high_bound'])\n",
+    "print('low_bound: ', box_plot_dict['low_bound'])\n",
+    "print('quantiles: ', box_plot_dict['quantiles'])\n",
+    "print('number of low outliers: ', len(box_plot_dict['low_values']))\n",
+    "print('number of high outliers: ', len(box_plot_dict['high_values']))"
```

Comment:
If I'm reading this correctly, the output is suggesting we have 1922 low outliers and 31016 high outliers. This seems like a lot of outliers - over 8% of the total data. Is this expected?

Reply:
Looking at all of the numeric columns in the retail dataframe, they all have somewhat tight quartiles, but a big enough standard deviation that the number of outliers makes some sense to me. But I think we don't necessarily know what would be expected in the data here. 

This does, however, make me want to confirm that the results for a normally distributed column of data are expected. Would it make sense to add that instead of (or maybe in addition to) the retail example here?

Reply:
Added a section that creates a normally distributed column of data the same size as the retail dataset, and the number of outliers pretty closely matches what we'd expect from normally distributed data (http://www.cs.uni.edu/~campbell/stat/normfact.html), so I'm guessing it's jus that the retail data isn't normally distributed.